Unscheduled Care/2. Unscheduled Care outputs.R

```diff
@@ -34,15 +34,15 @@ library(reshape2)
 # LOCALITY <- "Ayr North and Former Coalfield Communities"
 # LOCALITY <- "Whalsay and Skerries"
 # LOCALITY <- "North Perthshire"
-
+#LOCALITY <- "Inverclyde East"
 # Set date limit for financial year
 # Unless we're in Q4 use the previous FY as the max
 max_fy <- ifelse(
```

Comment:
I'm not sure whether I've commented on this before hmmmmmm. Would it be better having the max_fy as a function in Global?